raft/log_test.go

```diff
@@ -281,9 +281,7 @@ func TestLog_WriteEntriesTo_Snapshot(t *testing.T) {
 	}()
 
 	// Wait for buffer to be written to.
-	for buf.Len() == 0 {
-		runtime.Gosched()
-	}
+	time.Sleep(1 * time.Second)
```

Comment:
Why make this change? `Len()` has been locked, so there is no problem spinning on it, right?